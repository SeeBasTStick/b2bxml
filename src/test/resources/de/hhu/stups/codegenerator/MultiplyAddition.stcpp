int main() {
    Multiply multiply;
    multiply.calculate();
    cout << multiply.getRes();
    return 0;
}