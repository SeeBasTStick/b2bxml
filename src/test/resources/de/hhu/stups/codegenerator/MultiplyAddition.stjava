public static void main(String[] args) {
    Multiply multiply = new Multiply();
    multiply.calculate();
    System.out.println(multiply.getRes());
}