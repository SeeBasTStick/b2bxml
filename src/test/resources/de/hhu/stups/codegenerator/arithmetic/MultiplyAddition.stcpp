int main() {
    Multiply arithmetic;
    arithmetic.simulate();
    cout << arithmetic.getValue();
    return 0;
}