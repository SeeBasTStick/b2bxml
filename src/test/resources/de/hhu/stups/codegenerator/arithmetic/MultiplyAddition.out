public static void main(String[] args) {
    Multiply arithmetic = new Multiply();
    arithmetic.simulate();
    System.out.println(arithmetic.value);
}